.github/workflows/prune-images.yml

```diff
@@ -0,0 +1,63 @@
+name: Prune Images
+run-name: Prune old images from OpenShift
+
+on:
+  workflow_dispatch:
+    inputs:
+      applications:
+        description: 'Applications to prune'
+        required: true
+        default: 'web-sims, api-sims'
+      licensePlate:
```

Comment:
For every Github environment, we have available a variable that represents the Openshift namespace. This is consumed in other GitHub Actions. Can the `licensePlate` be replaced by this one instead, which means that it does not need to be provided as an input parameter and it would be "connected" to the select env.
```
NAMESPACE: ${{ secrets.OPENSHIFT_ENV_NAMESPACE }}
```